backend/src/main/java/com/cruru/applyform/service/ApplyFormService.java

```diff
@@ -91,14 +100,57 @@ private void validatePersonalDataCollection(ApplyFormSubmitRequest request) {
 
     private void saveAnswerReplies(AnswerCreateRequest answerCreateRequest, Applicant applicant) {
         for (String reply : answerCreateRequest.replies()) {
-            Question question = getQuestionById(answerCreateRequest.questionId());
+            Question question = questionRepository.findById(answerCreateRequest.questionId())
+                    .orElseThrow(QuestionNotFoundException::new);
             Answer answer = new Answer(reply, question, applicant);
             answerRepository.save(answer);
         }
     }
 
-    private Question getQuestionById(long questionId) {
-        return questionRepository.findById(questionId)
-                .orElseThrow(QuestionNotFoundException::new);
+    public ApplyFormResponse read(long applyFormId) {
+        ApplyForm applyForm = applyFormRepository.findById(applyFormId)
+                .orElseThrow(ApplyFormNotFoundException::new);
+
+        List<QuestionResponse> responses = questionRepository.findAllByApplyFormId(applyFormId)
+                .stream()
+                .map(this::toQuestionResponse)
+                .toList();
+
+        return new ApplyFormResponse(
+                applyForm.getTitle(),
+                applyForm.getDescription(),
+                applyForm.getStartDate(),
+                applyForm.getEndDate(),
+                responses
+        );
+    }
+
+    private QuestionResponse toQuestionResponse(Question question) {
+        List<Choice> choices = getChoices(question);
+        return new QuestionResponse(
+                question.getId(),
+                question.getQuestionType().name(),
+                question.getContent(),
+                question.getDescription(),
+                question.getSequence(),
+                toChoiceResponses(choices)
+        );
+    }
+
+    private List<Choice> getChoices(Question question) {
+        if (!question.hasChoice()) {
+            return Collections.emptyList();
+        }
+        return choiceRepository.findAllByQuestionId(question.getId());
+    }
```

Comment:
```suggestion
    private List<Choice> getChoices(Question question) {
        if (question.hasChoice()) {
            return choiceRepository.findAllByQuestionId(question.getId());
        }
        return Collections.emptyList();
    }
```

아주 사소하지만 요렇게 하면 가독성이 더 좋아질 것 같은데 어떨까요